test/utils/jwt/src/index.ts

```diff
@@ -12,8 +12,7 @@ const proposalsPath = `${process.env.KMS_WORKSPACE}/proposals`;
 const privateKeyPath = `${process.env.KMS_WORKSPACE}/private.pem`;
 const certificatePath = `${process.env.KMS_WORKSPACE}/cert.pem`;
 const kid = "Demo IDP kid";
-const hostPort = 3000;
-const host = `http://localhost:${hostPort}`;
+const host = `http://localhost:0`;
```

Comment:
with a fixed host, how will CCF distinguish between different issuers:
await Proposals.issuerProposal(host, proposalsPath);

We will need to add the the unique id because otherwise different proposals will just update the existing proposal, or is this intentional?

Reply:
I don't fully understand your question, we've moved **from** a fixed host **to** a dynamic one in this change. Regarding CCF, how we register the issuer is a topic of discussion in your other PR, my preference as discussed [here](https://github.com/microsoft/azure-privacy-sandbox-kms/pull/260#discussion_r1886607870) is that we just pass the address of the issuer which will as of this change be unique per issuer (at least locally)

Reply:
Let me clarify. When you register two issuers as localhost:0, for CCF this is the same issuer. Your two issuers have different keys, so validation of token from the first issuer will fail because CCF will use the key of the second to validate

Reply:
We won't register it as localhost:0, we'll use the actual address including the resolved port

Reply:
in index,ts localhost:0 is registered as issuer for all issuers
// generate issuer proposal
    await Proposals.issuerProposal(host, proposalsPath);

Reply:
> in index,ts localhost:0 is registered as issuer for all issuers // generate issuer proposal await Proposals.issuerProposal(host, proposalsPath);

Dom and I agreed this line needs to be removed because it is no longer needed. The registration of issuers is done with the correct port in scripts. Dom will do a seperate PR to remove the line

Reply:
Here's the separate PR to clean up old code:

https://github.com/microsoft/azure-privacy-sandbox-kms/pull/283